addons/headless/XEH_preInit.sqf

```diff
@@ -12,8 +12,7 @@ if (isServer) then {
     GVAR(headlessClients) = [];
     GVAR(inRebalance) = false;
     GVAR(endMissionCheckDelayed) = false;
-    GVAR(blacklistType) = [BLACKLIST_UAV];
```

Comment:
I didn't see anything about this being API. Is it API though?

Reply:
No, only object/group variable `acex_headless_blacklist` is API.